.github/workflows/continuous-integration.yml

```diff
@@ -17,10 +17,15 @@ jobs:
     env:
       DOCKER_BUILDKIT: 1
       CI: 1
+      TIPPECANOE_VERSION: ${{ matrix.version }}
+      VARIANT: ${{ matrix.variant }}
     steps:
       - uses: actions/checkout@v2
 
       - run: ./scripts/cibuild
+
+      - run: ./scripts/cipublish
+        if: github.ref == 'refs/heads/master'
```

Comment:
This `if` gates this step to builds associated with `master`. Technically, I could have:

- Created a separate job specifically for this release and gate it there
- Created a separate YAML file that only listens to `master` for release workflows

I opted for this approach because it seemed to fit the scope of the problem best (the build is relatively simple).

Reply:
Chiming in with some observations I made in https://github.com/azavea/loam/pull/38#pullrequestreview-388620210! This pattern of gating steps works well for Docker images like this or for libraries where we create multiple build artifact variants.

Splitting release-related work out into a separate workflow becomes more appealing when there's a single artifact to deploy, like an application, or a release step that encompasses all variant artifacts, like a Git tagging process.